<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqtDataSourceDialog</class>
 <widget class="QDialog" name="pqtDataSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Sources</string>
  </property>
  <widget class="QListView" name="listView_data_source_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>20</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data source name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_data_source_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Data source type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_data_source_type">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_host"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>AET</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_aet"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Spare edit field</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_spare">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>571</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_new">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>released()</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>released()</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>pushbutton_save_released()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_new</sender>
   <signal>released()</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>pushbutton_new_released()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete</sender>
   <signal>released()</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>pushbutton_delete_released()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_data_source_list</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>listview_data_source_activated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_data_source_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pqtDataSourceDialog</receiver>
   <slot>listview_data_source_activated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushbutton_save_released()</slot>
  <slot>pushbutton_new_released()</slot>
  <slot>pushbutton_delete_released()</slot>
  <slot>listview_data_source_activated(QModelIndex)</slot>
 </slots>
</ui>
